cinema/serializers.py

```diff
@@ -30,10 +30,21 @@ class Meta:
         fields = ("id", "name", "rows", "seats_in_row", "capacity")
 
 
+class MovieImageSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = ["id", "image"]
+
+
 class MovieSerializer(serializers.ModelSerializer):
+    image = serializers.ImageField(read_only=True)
+
     class Meta:
         model = Movie
-        fields = ("id", "title", "description", "duration", "genres", "actors")
+        fields = (
+            "id", "title", "description", "duration",
+            "image", "genres", "actors"
+        )
 
 
 class MovieListSerializer(MovieSerializer):
```

Comment:
don't forget to include image field here after you remove it from MovieSerializer